frontend/src/metabase/visualizations/visualizations/LinkViz/LinkViz.unit.spec.tsx

```diff
@@ -201,6 +202,43 @@ describe("LinkViz", () => {
   });
 
   describe("entity links", () => {
+    it("wraps the dashboard's description", async () => {
+      const longDescription =
+        "This is a very very very very very very very very very very long dashboard description. Boy, I sure hope it does not overflow the screen when it's displayed in a tooltip. The only thing that would make it worse is if the text didn't wrap.";
+
+      const settings = {
+        link: {
+          entity: {
+            id: 1,
+            db_id: 20,
+            name: "Table Une",
+            model: "table",
+            description: longDescription,
+          },
+        },
+      } as LinkCardVizSettings;
```

Comment:
Please avoid type casting:

```suggestion
      const settings: LinkCardVizSettings = {
        virtual_card: createMockVirtualCard(),
        link: {
          entity: {
            id: 1,
            db_id: 20,
            name: "Table Une",
            model: "table",
            description: longDescription,
          },
        },
      };
```